docs/relnotes.md

```diff
@@ -18,16 +18,40 @@ Published on 2023-01.
 [`taipy-rest` 2.1](https://pypi.org/project/taipy-rest/2.1.0/) packages.
 
 
-#### New Features
+### New Features
 
-**`taipy-gui`**<br/>2.1.0
+<h6 style="font-size: 1.2em"><strong><code>taipy</code></strong></h6>
+2.1
 
+- Taipy and all its dependencies now support Python 3.11.
+
+<h6 style="font-size: 1.2em"><strong><code>taipy-gui</code></strong></h6>
+2.1.0
+
+- A security feature has been added: the file `.taipyignore`, located next to
+  the Python main file, can list the paths that you want to prevent access to.<br/>
+  See [issue #501](https://github.com/Avaiga/taipy-gui/issues/501) or
+  [this section](manuals/gui/configuration.md#protect-your-application-files) for
+  details.
+- Charts can use a [Decimator]() instance that cleverly filters data
+  points out to greatly improve performance.<br/>
+  See the [Decimator documentation]() for more details.
 - Charts now support polar, funnel, candlesticks and many other types of charts.<br/>
-  See the [chart control](../manuals/gui/viselements/chart) section for details.
+  See the [chart control](manuals/gui/viselements/chart.md) section for details.
+- Charts now support the dark theme automatically.
+- Tooltips can be set on individual cells.<br/>
```

Comment:
Cells -> table cells? Or mention that this is related to tables

Reply:
👍